src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -727,15 +745,6 @@ object EntityClient {
       case Closed => "closed"
     case Open, Closed
 
-  object SecurityTag {
-
-    def fromString(securityTagString: String): Option[SecurityTag] = securityTagString match {
-      case "open"   => Option(Open)
```

Comment:
I think it's fine, as now, you can't accidentally/purposely add/delete/modify the strings without editing the enum values. The only thing that confused me at first was the fact that we had an object called `SecurityTag` as it confuses IntelliJ; since it has the same name as the enum, IntelliJ thinks that the `SecurityTag` in `Try(SecurityTag.valueOf(securityTagString)).toOption` is referring to itself (the object) and not the enum (The compiler is smart enough but not IntelliJ).